### PR TITLE
fixed/cleaned raw3-3 config

### PR DESCRIPTION
**Pull request by benmaidel**
removed old unnecessary frida stuff

### CODE REVIEW

**Comment by benmaidel** [comment]
tested on raw3-3!

**Comment by floweisshardt** [comment]
LGTM

**Comment by NadiaHG** [comment]
LGTM

File: cob_hardware_config/raw3-3/config/robot_modules.yaml [hunk 1]
```diff
@@ -1 +1 @@
-robot_modules: ['base','arm']
+robot_modules: ['base','torso']
```
Comment (mgruhler) [inline cob_hardware_config/raw3-3/config/robot_modules.yaml]: What is the torso on raw3-3? The linear actuators on top? Can this be moved via teleop (this is what `robot_modules.yaml` is for, right?) or why do we need to add this here?

Comment (benmaidel) [inline cob_hardware_config/raw3-3/config/robot_modules.yaml]: In fact a mistake. Mixed raw3-1 and raw3-3 together.

**Comment by fmessmer** [comment]
Formal Review OK

**Pull request merged**